tests/numpy/preprocessing/test_base.py

```diff
@@ -25,6 +25,9 @@ class TestStandardScaler:
 
     rng = np.random.default_rng(0x122578748CFF12AD12)
     n_strucs = 5
+
+    # this computation is only needed because equistore does not offer random
+    # initialization TODO change as soons as equistore as random initialization
```

Comment:
What do you mean with random initialization? something like `np.rand((3, 4))`? I doubt we will be able to provide something like this, since we also need metadata =/ You could have an helper function here that create a tensormap with known metadata and random data, in the same way this works: https://github.com/lab-cosmo/equistore/blob/master/python/tests/utils.py#L6

Reply:
Both things seems useful. For now I would just use an adapted version of the file you linked here, and put it in a test utils foder as you did in equstore.